contracts/feature-tests/basic-features/mandos/send_esdt.scen.json

```diff
@@ -0,0 +1,116 @@
+{
+    "name": "send_esdt",
+    "steps": [
+        {
+            "step": "setState",
+            "accounts": {
+                "address:features_contract": {
+                    "nonce": "1000",
+                    "balance": "0",
+                    "storage": {},
+                    "esdt": {
+                        "str:BASIC-FEATURES-TOKEN": "1,000"
+                    },
+                    "code": "file:../output/basic-features.wasm"
+                },
+                "address:an_account": {
+                    "nonce": "0",
+                    "balance": "0",
+                    "storage": {},
+                    "code": ""
+                }
+            }
+        },
+        {
+            "step": "scCall",
+            "txId": "not-enough",
+            "tx": {
+                "from": "address:an_account",
+                "to": "address:features_contract",
+                "value": "0",
+                "function": "send_esdt",
+                "arguments": [
+                    "address:an_account",
+                    "str:BASIC-FEATURES-TOKEN",
+                    "10000"
+                ],
+                "gasLimit": "0x100000",
+                "gasPrice": "0"
+            },
+            "expect": {
+                "out": [],
+                "status": "*",
+                "message": "str:insufficient funds",
+                "logs": []
+            }
+        },
+        {
+            "step": "scCall",
+            "txId": "without-message",
+            "tx": {
+                "from": "address:an_account",
+                "to": "address:features_contract",
+                "value": "0",
+                "function": "send_esdt",
+                "arguments": [
+                    "address:an_account",
+                    "str:BASIC-FEATURES-TOKEN",
+                    "100"
+                ],
+                "gasLimit": "0x100000",
+                "gasPrice": "0"
+            },
+            "expect": {
+                "out": [],
+                "status": "",
+                "logs": []
+            }
+        },
+        {
+            "step": "scCall",
+            "txId": "with-message",
+            "tx": {
+                "from": "address:an_account",
+                "to": "address:features_contract",
+                "value": "0",
+                "function": "send_esdt",
```

Comment:
There are two calls for `send_esdt` but none for `send_esdt_twice`.
I think `send_esdt_twice` should be tested as well.

Reply:
Only tested it on public testnet, not yet sure if both mandoses can handle it.